File: BaoCao01/app/src/main/java/com/example/baocao1/Model/TheLoai.java
package com.example.baocao1.Model;

public class TheLoai {
    private String MaTheLoai,TenTheLoai,HinhAnh;

    public String getMaTheLoai() {
        return MaTheLoai;
    }

    public void setMaTheLoai(String maTheLoai) {
        MaTheLoai = maTheLoai;
    }

    public String getTenTheLoai() {
        return TenTheLoai;
    }

    public void setTenTheLoai(String tenTheLoai) {
        TenTheLoai = tenTheLoai;
    }

    public String getHinhAnh() {
        return HinhAnh;
    }

    public void setHinhAnh(String hinhAnh) {
        HinhAnh = hinhAnh;
    }

    public TheLoai(String maTheLoai, String tenTheLoai, String hinhAnh) {
        MaTheLoai = maTheLoai;
        TenTheLoai = tenTheLoai;
        HinhAnh = hinhAnh;
    }
}

File: Web/BookHeavenStoreMVC/Models/TheLoai.cs
using System.ComponentModel.DataAnnotations;

namespace BookHeavenStoreMVC.Models
{
    public class TheLoai
    {
        [Key]
        public string MaTheLoai {  get; set; }
        [Required]
        public string TenTheLoai { get; set; }
        public string? HinhAnh {  get; set; }
    }
}
